kotlinpoet-km/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=kotlinpoet-km
```

Comment:
Don't like this name here or in the package or the type names. Took a while to figure out what it meant.

Reply:
Honestly I don't know what a good option is. I'd be fine with `kotlinpoet-metadata`, but also would be curious to know if we can figure out the name separately as a followup to reduce churn from the implementation

Reply:
I do feel strongly about them being in the type names though I think, as the real types are prefixed with that and these are just immutable versions of them